### PR TITLE
Feature/ms sql

### PR DESCRIPTION
**Pull request by juliantopley**

### CODE REVIEW

File: _docs/applications/mssql.md [hunk 1]
```diff
@@ -1,36 +1,40 @@
 ---
 layout: guide
-title: StorageOS Docs - Microsoft SQL Server
+title: Microsoft SQL Server with StorageOS
```
Comment (cjh84) [inline _docs/applications/mssql.md]: Revert this. The title should be consistent across all docs.

File: _docs/applications/mssql.md [hunk 2]
```diff
@@ -1,36 +1,40 @@
...
 
 # ![image](/images/docs/explore/mssqllogo.png) Microsoft SQL Server 2016 with StorageOS
 
- Microsoft's latest SQL Server release represents a major update with availability for Linux platform support as well support to run as a container.
+ Microsoft's latest SQL Server release represents a major update with availability for Linux platform support and support to run as a container.
 
 ## Linux Support
 
-For Ubuntu, the Docker container requires Xenial 16.04.
+Microsoft state SQL Server for Linux runs with any supported Linux distribution.  To date we have only tested with Ubuntu 16.04 which we used for this document and Alpine Linux (which did not work).  Additional requirements include:
```
Comment (cjh84) [inline _docs/applications/mssql.md]: "To date we have only tested with Ubuntu 16.04 which we used for this document and Alpine Linux (which did not work)."

Change this to "The recommended distro for testing StorageOS is Ubuntu 16.04."

File: _docs/applications/mssql.md [hunk 3]
```diff
@@ -1,36 +1,40 @@
...
+* Minimum of 4 GB of disk space 
+* Minimum of 4 GB of RAM (has been tested to work with 3.25GB)
+
+Starting with SQL Server vNext CTP 1.4, the SQL Server command-line tools are included in the Docker image. If you attach to the image with an interactive command-prompt, you can run the tools locally.
```
Comment (cjh84) [inline _docs/applications/mssql.md]: Remove vNext, it's the project name. MS only uses SQL Server.

File: _docs/applications/mssql.md [hunk 4]
```diff
@@ -1,36 +1,40 @@
...
 
 ## More Information
 
-More information on SQL Server for Linux is available on the [Microsoft SQL Server for Linux](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-overview "Microsoft SQL Server for Linuxy") web page.
+More information on SQL Server for Linux is available on the [Microsoft SQL Server for Linux](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-overview "Microsoft SQL Server for Linux") web page.
 
-Before you start, please ensure you have StorageOS installed and ready on a Linux cluster - please refer to the *Installation Guide* for further details.
+Before you start, please ensure you have StorageOS installed and ready on a Linux cluster - please refer to the [Cluster install ](../install/clusterinstall.html)section for further details.
```
Comment (cjh84) [inline _docs/applications/mssql.md]: Remove "please" (see the other guides).

File: _guides/004-applications.md [hunk 5]
```diff
@@ -12,4 +12,8 @@ sections:
     title: PostgreSQL
     description: Setting up PostgreSQL with persistent storage.
     module: applications/postgres
+  -
+    title: Microsooft SQL
```
Comment (cjh84) [inline _guides/004-applications.md]: Typo "Microsooft" => "Microsoft".

File: _docs/applications/mssql.md [hunk 6]
```diff
@@ -42,52 +46,28 @@ Before you start, please ensure you have StorageOS installed and ready on a Linu
...
 
-   ```
-   $ echo 'export PATH="$PATH:/opt/mssql-tools/bin"' >> ~/.bash_profile
-   $ echo 'export PATH="$PATH:/opt/mssql-tools/bin"' >> ~/.bashrc
-   $ source ~/.bashrc
-   ```
+1. Connect to SQL Server
 
-## Create a Test Database
+   >**Note**: We are using *localhost* in this example - if you are running this from a remote host you will need to specify the target server IP.
```
Comment (cjh84) [inline _docs/applications/mssql.md]: According to https://docs.microsoft.com/en-us/sql/tools/sqlcmd-utility, -S localhost is the default. If it's not needed then remove it and move the note to line 104.

File: _docs/applications/mssql.md (hunk 2 above)
Comment (cjh84) [inline _docs/applications/mssql.md]: Remove "state".

File: _docs/applications/mssql.md [hunk 7]
```diff
@@ -1,36 +1,40 @@
...
 ## Linux Support
 
-For Ubuntu, the Docker container requires Xenial 16.04.
+Microsoft state SQL Server for Linux runs with any supported Linux distribution.  To date we have only tested with Ubuntu 16.04 which we used for this document and Alpine Linux (which did not work).  Additional requirements include:
+* Minimum of 4 GB of disk space 
+* Minimum of 4 GB of RAM (has been tested to work with 3.25GB)
+
+Starting with SQL Server vNext CTP 1.4, the SQL Server command-line tools are included in the Docker image. If you attach to the image with an interactive command-prompt, you can run the tools locally.
 
 ## More Information
```
Comment (cjh84) [inline _docs/applications/mssql.md]: Remove this section and move the More information sentence to the bottom of the page.

File: _docs/applications/mssql.md [hunk 8]
```diff
@@ -42,52 +46,28 @@ Before you start, please ensure you have StorageOS installed and ready on a Linu
...
-       on Linux (Ubuntu 16.04.1 LTS)
+   ...
+   2017-04-13 09:24:22.28 Server      Dedicated admin connection support was established for listening locally on port 1434.
+   2017-04-13 09:24:22.28 spid17s     SQL Server is now ready for client connections. This is an informational message; no user action is required.
    ```
 
-## Install MS SQL Tools on Ubuntu
-
-Sqlcmd is part of the SQL Server command-line tools, which are not installed automatically with SQL Server on Linux and not present in the container image.
+## Install SQL Server command-line tools on Linux
```
Comment (cjh84) [inline _docs/applications/mssql.md]: Since Sqlcmd is already included, remove this entire section.

File: _docs/applications/mssql.md [hunk 9]
```diff
@@ -1,36 +1,40 @@
...
-More information on SQL Server for Linux is available on the [Microsoft SQL Server for Linux](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-overview "Microsoft SQL Server for Linuxy") web page.
+More information on SQL Server for Linux is available on the [Microsoft SQL Server for Linux](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-overview "Microsoft SQL Server for Linux") web page.
 
-Before you start, please ensure you have StorageOS installed and ready on a Linux cluster - please refer to the *Installation Guide* for further details.
+Before you start, please ensure you have StorageOS installed and ready on a Linux cluster - please refer to the [Cluster install ](../install/clusterinstall.html)section for further details.
 
 ## Install MS SQL Server with StorageOS
 
-1. Get the latest MS SQL Server Linux container up and running
+1. Startup the latest MS SQL Server container on a StorageOS node with a StorageOS persistent volume
```
Comment (cjh84) [inline _docs/applications/mssql.md]: Add a colon to the end of the sentence.

Throughout this doc, Instructions to a user to run a command should end in a colon.

File: _docs/applications/mssql.md (hunk 7 above)
Comment (juliantopley) [inline _docs/applications/mssql.md]: Ah - you want to move this to the More information section *and* move the section to the bottom of the page.

File: _docs/applications/mssql.md [hunk 10]
```diff
@@ -186,3 +150,11 @@ Sqlcmd is part of the SQL Server command-line tools, which are not installed aut
...
    1> quit
    ```
+
+## More Information
+
+Starting with SQL Server 2017 CTP 2.0, the SQL Server command-line tools are included in the Docker image. If you attach to the image with an interactive command-prompt, you can run the tools locally.
+
+More information on SQL Server for Linux is available on the [Microsoft SQL Server for Linux](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-overview "Microsoft SQL Server for Linux") web page.
+
+Before you start, ensure you have StorageOS installed and ready on a Linux cluster - please refer to the [Cluster install ](../install/clusterinstall.html)section for further details.
```
Comment (cjh84) [inline _docs/applications/mssql.md]: Move "Before you start..." to line 18.

Comment (juliantopley) [inline _docs/applications/mssql.md]: I thought that might want to move back up :-)

**Pull request merged**